certbot-nginx/certbot_nginx/parser.py

```diff
@@ -312,6 +313,32 @@ def add_server_directives(self, vhost, directives, replace):
         except errors.MisconfigurationError as err:
             raise errors.MisconfigurationError("Problem in %s: %s" % (filename, str(err)))
 
+    def create_new_vhost_from_default(self, vhost_template):
+        """Duplicate the default vhost in the configuration files.
+
+        :param :class:`~certbot_nginx.obj.VirtualHost` vhost_template: The vhost
+            whose information we copy
+
+        :returns: A vhost object for the newly created vhost
+        :rtype: :class:`~certbot_nginx.obj.VirtualHost`
+        """
+        # TODO: do the right thing for debian
+        # put it in the same file as the template, at the same level
+        enclosing_block = self.parsed[vhost_template.filep]
+        for index in vhost_template.path[:-1]:
+            enclosing_block = enclosing_block[index]
```

Comment:
nit: there's no coverage on this line. It'd be nice to have, but it's not a big deal so feel free to ignore this comment if you'd like.

Reply:
I'll check if there's already a file with a structure that would allow testing this and if so I'll add a test, my thought before was that changing around all the testing files as a result of creating a test case for it which is how Nginx tests work seemed not worth it.